lit/SwiftExpr/BreakpointREPL.test

```diff
@@ -0,0 +1,13 @@
+// Test that we can set breakpoints in the REPL.
+
+// RUN: %lldb --repl < %s | FileCheck %s
+
+func foo() -> Int {
+  return 0
+}
+
+:b foo
```

Comment:
You switched this from a file & line breakpoint in the original test to a function breakpoint.  It would be fine to add a function name test but you shouldn't remove the test for file & line breakpoints.

Reply:
That's the hard part, after all - making sure that subsequent REPL invocations keep a useful line number mapping...

Reply:
This is a new test, it's not meant to replace anything (I mentioned this in the PR description). It's just a basic sanity check.